### PR TITLE
feat: Change harvest sentry url to errors.cozycloud.cc

### PR DESCRIPTION
**Pull request by doubleface**

### CODE REVIEW

File: packages/cozy-harvest-lib/src/sentry.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as Sentry from '@sentry/browser'
  * @param {SentryClient}
  */
 export const client = new Sentry.BrowserClient({
-  dsn: 'https://888abd94993a47a39e751598fc6be803@sentry.cozycloud.cc/145',
+  dsn: 'https://1f8d191bad884bf98ae8cdcb6f6abb72@errors.cozycloud.cc/39',
```
Comment (trollepierre) [inline packages/cozy-harvest-lib/src/sentry.js:9]: it is normal to set public key inside repopsitory? 


shouldn't we use at least variable?

Comment (sblaisot) [reply packages/cozy-harvest-lib/src/sentry.js:9]: all responses to your concerns are in sentry documentation: https://docs.sentry.io/product/sentry-basics/dsn-explainer/#dsn-utilization

TL;DR: No major problem having it there.

**Review by trollepierre — APPROVED** [review]

**Comment by Crash--** [comment]
ping @doubleface
